composer.json

```diff
@@ -12,6 +12,9 @@
       "url": "https://repo.magento.com/"
     }
   ],
+  "require": {
+    "mage-os/mageos-async-events": "*"
```

Comment:
Please add missing modules here

Reply:
We don't have additional dependencies. I'd rather not add magento/sales and magento/customer here, because the module will work flawlessly if we don't have those modules. I'll add them to the module.xml as a soft dependency though.